subsys/bluetooth/controller/Kconfig

```diff
@@ -494,6 +503,13 @@ config BT_CTLR_ADV_PERIODIC
 	bool "LE Periodic Advertising in Advertising State [EXPERIMENTAL]" if BT_LL_SW_SPLIT
 	select EXPERIMENTAL if BT_LL_SW_SPLIT
 
+config BT_CTLR_ADV_PERIODIC_ADI_SUPPORT
+	bool "Periodic Advertising ADI support"
```

Comment:
May be worthwhile to spell out ADI fully here if someone is not familiar with the acronym.